translations/en-us.json

```
@@ -72,8 +72,8 @@
   "contact_us_button_text_description": "Text for New Request Button",
   "mit_address_description": "MIT address",
   "mit_address_label": "Address",
-  "mit_home_description": "MIT homepage page URL",
-  "mit_home_label": "Homepage",
+  "mit_ol_home_description": "MIT Open Learning homepage page URL",
+  "mit_ol_home_label": "Homepage",
```

Maybe this should be?

```suggestion
  "mit_ol_home_label": "MIT Open Learning Homepage",
```

---

Fixed in https://github.com/mitodl/micromasters-zendesk-theme/pull/1/commits/c588df8a6399e48407c0f8014ed99f9ec382ee4b